pytorch/training/docker/1.12/py3/cu113/Dockerfile.gpu

```diff
@@ -180,7 +180,7 @@ RUN ompi_info --parsable --all | grep mpi_built_with_cuda_support:value \
     conda-content-trust \
     charset-normalizer \
     "shap<1" \
- && /opt/conda/bin/conda install -c dglteam -y dgl-cuda11.3 \
+ && /opt/conda/bin/conda install -c dglteam -y dgl-cuda11.3=0.9.1 \
```

Comment:
Why are we specifying version?

Reply:
They had a recent release of 0.9.1post1, but the upstream test has caught up and failed. 
This morning when I tried the test it seemed to work fine. Trying a build with dgl unpinned.